autogpt_platform/frontend/src/hooks/useAgentGraph.tsx

```diff
@@ -159,96 +150,93 @@ export default function useAgentGraph(
   );
 
   // Load existing graph
-  const loadGraph = useCallback(
-    (graph: Graph) => {
-      setSavedAgent(graph);
-      setAgentName(graph.name);
-      setAgentDescription(graph.description);
-
-      setXYNodes((prevNodes) => {
-        const _newNodes = graph.nodes.map((node) => {
-          const block = availableBlocks.find(
-            (block) => block.id === node.block_id,
-          )!;
-          if (!block) return null;
-          const prevNode = prevNodes.find((n) => n.id === node.id);
-          const flow =
-            block.uiType == BlockUIType.AGENT
-              ? availableFlows.find(
-                  (flow) => flow.id === node.input_default.graph_id,
-                )
-              : null;
-          const newNode: CustomNode = {
-            id: node.id,
+  function _loadGraph(graph: Graph) {
```

Comment:
No changes inside this function, I just renamed it to make sure that we don't have to include it as a dependency in hooks where it's used, and so it's clear that this is not exported.